README.rst

```diff
@@ -615,6 +615,10 @@ Choose from the list of available fixers:
                         construct to print if
                         possible.
 
+* **empty_phpdoc** [contrib]
```

Comment:
I'll ask them if they reply on my PR overthere, till than I'm not going to the trouble to check it all out. If they want it we can always move it to SF level.